### PR TITLE
Better border-box reset

### LINKED ISSUE #10: suggestion: better border-box reset?
Just a small contribution.

I checked your nice lib and found this:
https://github.com/mike-engel/a11y-css-reset/blob/13fa26d0fa978443c83c624f518f7438f0b0565c/reset.css#L4

About setting border-box everywhere, I heard about this technique which I believe is superior:

```css
/* apply a natural box layout model to all elements, but allowing components to change
 * https://www.paulirish.com/2012/box-sizing-border-box-ftw/
 */
html                 { box-sizing: border-box; }
*, *:before, *:after { box-sizing: inherit; }
```

The rationale is that it allows to opt-out easily if needed.

### PR DESCRIPTION
**Pull request by Offirmo**
fixes #10

### CODE REVIEW

File: reset.css [hunk 1]
```diff
@@ -1,8 +1,8 @@
-*,
-*::before,
-*::after {
-  box-sizing: border-box;
-}
+/* apply a natural box layout model to all elements, but allowing components to change
+ * https://www.paulirish.com/2012/box-sizing-border-box-ftw/
+ */
+html                 { box-sizing: border-box; }
+*, *:before, *:after { box-sizing: inherit; }
```
Comment (mike-engel) [inline reset.css]: To match the old code, can you ensure that `before` and `after` blocks use two colons? See #6 for more info

**Review by mike-engel — CHANGES_REQUESTED** [review]

File: reset.css (hunk 1 above)
Comment (Offirmo) [reply reset.css]: Wow I didn't know that.

**Comment by mike-engel** [comment]
@Offirmo Do you have time to make the edits from `:` to `::`, or would you like me to do that?

**Comment by Offirmo** [comment]
@mike-engel  sorry I have no time. Too bad I can't quick edit a PR...

**Comment by mike-engel** [comment]
Published as `1.2.0`. Thanks again!

**Pull request merged**